### PR TITLE
Updated descriptions about headers

### PR DESCRIPTION
**Pull request by justineechen**
Addressing https://github.com/OpenLiberty/guide-cors/issues/29

### CODE REVIEW

**Comment by justineechen** [comment]
- May need SME/ID review

**Comment by gkwan-ibm** [comment]
- [ ] format TestCors.java

**Comment by gkwan-ibm** [comment]
-------------------
Origin:
The value is set to `openliberty.io`. Indicates that the request originates from `openliberty.io`.


-------------------
Access-Control-Allow-Origin:
The expected value is `openliberty.io`. Indicates wheather a resource can be shared based by returning the value of the Origin request header `openliberty.io`. 

Access-Control-Allow-Credentials:
The expected value is `true`. Indicates that the user credentials can be included in the request.

Access-Control-Expose-Headers:
The expected value is `MyHeader`. Indicates that the header `MyHeader` is safe to expose.


-------------------
Origin:
The value is set to `anywebsiteyoulike.com`. Indicates that the request originates from `anywebsiteyoulike.com`.

Access-Control-Request-Method:
The value is set to DELETE. Indicates that HTTP DETLETE method will be used in the actual request.

Access-Control-Request-Headers:
The value is set to `MyOwnHeader2`. Indicates the header `MyOwnHeader2` will be used in the actual request.


-------------------
Access-Control-Allow-Max-Age:
The expected value to be 10. Indicates that the preflight request can be cached within 10 seconds.

Access-Control-Allow-Origin:
The expected value is `anywebsiteyoulike.com`. Indicates wheather a resource can be shared based by returning the value of the Origin request header `anywebsiteyoulike.com`.

Access-Control-Allow-Methods:
The expected value is `OPTIONS, DELETE`. Indicates that HTTP OPTIONS and DELETE methods can be used in the actual request.

Access-Control-Allow-Credentials:
The expected value is `true`. Indicates that the user credentials can be included in the request.

Access-Control-Expose-Headers:
The expected value is `MyOwnHeader1, MyOwnHeader2`. Indicates that the header `MyOwnHeader1` and `MyOwnHeader2` are safe to expose.

File: finish/src/test/java/it/io/openliberty/guides/cors/TestCors.java [hunk 1]
```diff
@@ -59,10 +59,12 @@ public void testPreflightCorsRequest() throws IOException {
     }
     // end::preflightTest[]
 
-    public void checkCorsResponse(HttpURLConnection connection, Map<String, String> expectedHeaders) throws IOException {
+    public void checkCorsResponse(HttpURLConnection connection,
```
Comment (proubatsis) [inline finish/src/test/java/it/io/openliberty/guides/cors/TestCors.java]: These changes should also be done in the `start/` directory.

File: README.adoc [hunk 2]
```diff
@@ -231,21 +245,21 @@ The [hotspot=36-44 file=1]`testPreflightCorsRequest` test simulates a client sen
...
-| *Header* | *Value*
-| Access-Control-Allow-Max-Age | [hotspot=23 file=0]`10`
-| Access-Control-Allow-Origin | anywebsiteyoulike.com
-| Access-Control-Allow-Methods | [hotspot=20 file=0]`OPTIONS, DELETE`
-| Access-Control-Allow-Credentials | [hotspot=21 file=0]`true`
-| Access-Control-Allow-Headers | [hotspot=22 file=0]`MyOwnHeader1, MyOwnHeader2`
+| *Response Header* | *Response Value*
+| Access-Control-Allow-Max-Age | The expected value is `10`. Indicates that the preflight request can be cached within `10` seconds.
+| Access-Control-Allow-Origin | The expected value is `anywebsiteyoulike.com`. Indicates whether a resource can be shared based on the returning value of the Origin request header `anywebsiteyoulike.com`.
+| Access-Control-Allow-Methods | The expected value is `OPTIONS, DELETE`.  Indicates that HTTP OPTIONS and DELETE methods can be used in the actual request.
```
Comment (proubatsis) [inline README.adoc]: There are two spaces after the period on this line.

Comment (justineechen) [reply README.adoc]: I don't see which line you mean

File: finish/src/test/java/it/io/openliberty/guides/cors/TestCors.java (hunk 1 above)
Comment (justineechen) [reply finish/src/test/java/it/io/openliberty/guides/cors/TestCors.java]: Yup! I can update that one

File: README.adoc (hunk 2 above)
Comment (justineechen) [reply README.adoc]: Removed extra spacing on Line 260!

**Pull request merged**